test/field_types/test_file_field.py

```diff
@@ -19,6 +22,14 @@ class ModelWithFile(MongoModel):
     secondary_upload = FileField(required=False)
 
 
+class ModelWithFileNoConnection(MongoModel):
+    upload = FileField()
+    secondary_upload = FileField(required=False)
+
+    class Meta:
+        connection_alias = 'test-field-file-no-connection'
```

Comment:
Can you define this model inside `test_lazy_storage_initialization`? That way only that test fails when/if we regress to the old behavior.